GiniSwitchSDK/GiniSwitchSDK/Classes/GiniSwitchLogger.swift

```diff
@@ -24,3 +24,9 @@ public protocol GiniSwitchLogger {
     func logError(message:String)
     
 }
+
+var logger:GiniSwitchLogger = GiniSwitchConsoleLogger()
+
+func Logger() -> GiniSwitchLogger {
```

Comment:
I think it is weird to have a function start with an upper case letter. Why not `getLogger()` and why is it even needed can't I just access `logger`?